GOVERNANCE.MD

```diff
@@ -1,6 +1,91 @@
+
+Section 8. Voting
+
+For internal project decisions, Collaborators shall operate under Lazy Consensus. The CC shall establish appropriate guidelines for implementing Lazy Consensus (e.g. expected notification and review time periods) within the development process.
+
+The CC follows a Consensus Seeking decision making model. When an agenda item has appeared to reach a consensus the moderator will ask "Does anyone object?" as a final call for dissent from the consensus.
+
+If an agenda item cannot reach a consensus a CC member can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be seconded by a majority of the CC or else the discussion will continue. Simple majority wins, with the following exceptions, which will require the affirmative vote of two-thirds of the members of the CC to pass:
+
+- Adding or removing members of the CC
+- Changes to the CC Charter (which also require Board approval)
+
+Section 9. Roles
+
+- The Node.js Foundation community git repository will be maintained by the CC and additional Collaborators who are added by the CC on an ongoing basis.
+- Individuals making significant and valuable contributions, “Contributor(s)”, are made Collaborators and given commit-access to the project. These individuals are identified by the CC and their addition as Collaborators is discussed during the weekly CC meeting. Modifications of the contents of the git repository are made on a collaborative basis as defined in the development process.
+- Collaborators may opt to elevate significant or controversial modifications, or modifications that have not found consensus to the CC for discussion by assigning the cc-agenda tag to a pull request or issue. The CC should serve as the final arbiter where required. The CC will maintain and publish a list of current Collaborators by Project, as well as a development process guide for Collaborators and Contributors looking to participate in the effort.
+
+Section 10. Definitions
+
+- Contributors: contribute code or other artifacts, but do not have the right to commit to the code base. Contributors work with the Project’s Collaborators to have code committed to the code base. A Contributor may be promoted to a Collaborator by the projects’ Maintainer or the CC. Contributors should rarely be encumbered by the CC and never by the Board.
+- Project: a collaboration effort, e.g. a subsystem, that is organized through the project creation process and approved by the CC.
+Maintainer: a Collaborator within a Core Project elected to represent the Core Project on the CC.
```

Comment:
Will the CC have Core Projects? Maybe if like NodeSchool is brought in to the foundation..? I'm, personally, would like to see the term removed (I've suggested it to the TSC's charter update we're working on)